src/MembershipsSDKAdapter.js

```diff
@@ -84,6 +93,7 @@ function sortRoomMembers(memberships, myID) {
  * @returns {Array.<Member>} List of meeting members
  */
 function getMembers(sdkMembers) {
+  logger.debug(...LOG_ARGS, 'getMembers()', ['called with', sdkMembers]);
```

Comment:
for consistency, use `'called with', {sdkMembers}` so that the property name is also shown (sdkMembers)